src/lib.rs

```diff
@@ -110,6 +110,21 @@
 //! Observe that the same code using, say [`signing_only`](struct.Secp256k1.html#method.signing_only)
 //! to generate a context would simply not compile.
 //!
+//! ## Crate features/optional dependencies
+//!
+//! The crate provides following opt-in Cargo features:
+//!
+//! * `std` - use standard Rust library, enabled by default.
+//! * `alloc` - use the `alloc` standard Rust library to provide heap allocations.
+//! * `rand` - use `rand` library to provide random generator (to e.g. generate keys)
+//! * `rand-std` - use `rand` library with its `std` feature enabled. (Implies `rand`.)
+//! * `recovery` - enable functions that can compute the public key from signature
+//! * `lowmemory` - optimize the library for low-memory environments
+//! * `global-context` - enable use of global secp256k1 context. (Implies `std`, `rand-std` and
+//!                      `global-context-less-secure`.)
+//! * `global-context-less-secure` - enables global context and opts-in to lower security.
+//! * `serde` - implements serialization and deserialization for types in this crate using `serde`
+//! * `bitcoin_hashes` - enables interaction with the `bitcoin-hashes` crate (e.g. conversions)
```

Comment:
```suggestion
//! * `serde` - implements serialization and deserialization for types in this crate using `serde`.
//! * `bitcoin_hashes` - enables interaction with the `bitcoin-hashes` crate (e.g. conversions).
```
Just hit me when you see me :)